test/cli/export-globals.test.js

```diff
@@ -1,20 +1,23 @@
 /* global describe, it, exec, expect */
-var fs = require('fs');
+var fs = require('fs'),
+    path = require('path'),
+
+    exportedGlobalsPath = path.join(__dirname, '..', '..', 'out', 'test-globals.json');
 
 describe('--export-globals', function () {
     this.timeout(1000 * 60); // set 60s timeout
 
     afterEach(function () {
-        try { fs.unlinkSync('out/test-globals.json'); }
+        try { fs.unlinkSync(exportedGlobalsPath); }
         catch (e) { console.error(e); }
     });
 
     it('`newman run` should export globals to a file', function (done) {
         // eslint-disable-next-line max-len
-        exec('./bin/newman.js run test/cli/single-get-request.json -g test/cli/simple-variables.json --export-globals out/test-globals.json', function (code) {
+        exec('node ./bin/newman.js run test/cli/single-get-request.json -g test/cli/simple-variables.json --export-globals out/test-globals.json', function (code) {
```

Comment:
yes. Thoughts? Only downside is that if we run anything other than scripts, it will get mucked up.

Reason I say this is because, while writing tests, people will not remember and when Windows test fail, then they'll add back.